docs/reference/modules/coordination.asciidoc

```diff
@@ -0,0 +1,461 @@
+    {"id":"2Z9lgUSpTgSUYaBSIKbJRG"}
+  ]
+}
+--------------------------------------------------
+
+It is safe to repeatedly call `GET /_cluster/bootstrap`, and to call it on
+different nodes concurrently. This API will yield an error if the receiving
+node has already been bootstrapped or has joined an existing cluster.
+
+Once a bootstrap document has been received, it must then be sent back to the
+cluster to finish the bootstrapping process as follows:
+
+[source,js]
+--------------------------------------------------
+# send the bootstrap document back to the cluster
+POST /_cluster/bootstrap
+{
+  "master_nodes_failure_tolerance": 1,
+  "master_nodes":[
+    {"id":"USpTGYaBSIKbgSUJR2Z9lg"},
+    {"id":"gSUJR2Z9lgUSpTGYaBSIKb"},
+    {"id":"2Z9lgUSpTgSUYaBSIKbJRG"}
+  ]
+}
+--------------------------------------------------
+// CONSOLE
+
+It is safe to repeatedly call `POST /_cluster/bootstrap`, and to call it on
+different nodes concurrently, but **it is vitally important** to use the same
+bootstrap document in each call.
+
+It is also possible to select the initial set of nodes in terms of their names
+rather than their IDs as follows.
+
+[source,js]
+--------------------------------------------------
+# send the bootstrap document back to the cluster
+POST /_cluster/bootstrap
+{
+  "master_nodes_failure_tolerance": 1,
+  "master_nodes":[
+    {"name":"master-a"},
+    {"name":"master-b"},
+    {"name":"master-c"}
+  ]
+}
+--------------------------------------------------
+// CONSOLE
+
+This can be useful if the node names are known (and known to be unique) in
+advance, and means that the first `GET /_cluster/bootstrap` call is not
+necessary. As above, it is safe to repeatedly call `POST /_cluster/bootstrap`,
+and to call it on different nodes concurrently, but **it is vitally important**
+to use the same bootstrap document in each call.
+
+[float]
+=== Manually-triggered elections
+
+It is possible to request that a particular node takes over from the elected
+master as follows:
+
+[source,js]
+--------------------------------------------------
+POST /_nodes/node_name/start_election
+--------------------------------------------------
+// CONSOLE
+
+Elections are not guaranteed to succeed, and a new leader may be elected at any
+time so even if this election does succeed then there may be another election,
+so there is no guarantee that the chosen node will be the elected master for
+any length of time.
+
+[float]
+=== Unsafe disaster recovery
+
+In a disaster situation a cluster may have lost half or more of its
+master-eligible nodes and therefore be in a state in which it cannot elect a
+master. There is no way to recover from this situation without risking data
+loss, but if there is no other viable path forwards then this may be necessary.
+This can be performed with the following command on a surviving node:
+
+[source,js]
+--------------------------------------------------
+POST /_nodes/_local/force_become_leader
```

Comment:
I'm not sure if this should be on the _nodes endpoint or whether it should be on an endpoint similar to `_cluster/bootstrap`.

Reply:
I struggled with naming, but yes I think so too. I changed this to

    POST /_cluster/force_local_node_takeover